docs/control-structures.rst

```diff
@@ -111,9 +112,9 @@ Solidity.
 Returning Multiple Variables
 ----------------------------
 
-When a function has multiple return types, ``return (v0, v1, ...,
-vn)`` can return multiple values.  The number of components must be
-the same as the number of return types.
+When a function has multiple return variable types, for example ``return (v0, v1, ..., vn)``,
```

Comment:
As is, this does not really work. This subsection is about the return statement (it is inside "control structures"), so the example does not clarify the multiple return variables of the function but rather how to use a return statement with multiple values.

Reply:
@chriseth You mean the existing example code? I think I mentioned in https://github.com/ethereum/solidity/issues/5425 that some better examples here might be useful, just want to check if we're talking about the same thing.